### DISCUSSION TITLE
Stream keeps switching on, non-stop

### DISCUSSION THREAD

**Discussion by yandemontreal** [tsightler/ring-mqtt#648] [Support Q&A] [answered]
It started happening last Friday and I don't recall updating anything except maybe HA Core. I for sure didn't change any configuration. Now in my Ring account, I have non-stop 12 minutes recordings. I had to stop ring-mqtt to prevent this. I don't want to be flagged by Jeff but also this keeps me from detecting motion in automations, and more. 

So today I updated the ring-mqtt addon and followed the instructions to deauthorize the third party apps, and reauthenticated. Still, the stream turns on right away, and if I manually switch it off, it starts again within 10 seconds. There isn't any movement out of the ordinary.

Surely it must be just me?

Ring doorbell pro 2

**Comment by tsightler** [comment, marked as answer]
If the stream is starting then _something_ is requesting it to start so you have to figure out what that is.  Probably easiest thing to do is to set/change the livestream username/password so that nothing can connect to it and then work through each configured client until you figure out what is keeping it alive.  It's almost always the preload stream setting in HA, but I've seen a few cases where some other addon or tool on the network that has autodiscovery, including HA itself, will discover the RTSP stream and keep triggering it.  Also, make sure you haven't exposed the RTSP port publicly if you don't need to use the stream outside of HA.

**Reply by yandemontreal** [reply]
Thank you for the prompt response. Following your explanation, I suspected go2rtc (in HA). With ring-mqtt-ha-addon stopped, I then stopped and restarted go2rtc, then started ring-mqtt-ha-addon. After this, it worked normally. Then I tried to restart Home Assistant OS to attempt to simulate the HA Core update/restart, but I was unable to reproduce the issue. 

So everything works now, and probably that a HA reboot would have sufficed in the first place. I had only tried a HA restart.
